dotrix_core/src/lib.rs

```diff
@@ -138,6 +138,9 @@ impl Dotrix {
         // Reset input events
         app.add_system(System::from(input::release));
 
+        // Reload and clean up assets
+        app.add_system(System::from(assets::assets_reload));
```

Comment:
Please, call the system `release`. First, this will move the system to `RunLevel::Release`, so it will be always executed when frame is rendered. Second, it will look more clear: `::from(assets::release)`